src/api/component.js

```diff
@@ -320,4 +320,8 @@ Component.prototype = Object.create(HTMLElement.prototype, {
   }
 });
 
+export const component = (superClass = HTMLElement) => {
+  return Component.extend({}, superClass);
```

Comment:
I'm not sure this would actually work, would it? It would extend the intended built-in, but because you've overwritten `this`, pointing to `Component` in the `extend()` function, it won't get Skate behaviour, would it?